src/GslCore/DnaCreation.fs

```diff
@@ -552,7 +515,28 @@ let expandAssembly
 
                 match ppp.part with
                 | MARKERPART ->
-                    yield expandMarkerPart library dnaSource ppp
+                    // Choose provider
+                    let providers =
+                        markerProviders |> 
+                            List.choose (fun provider -> 
+                                                    match provider.ScoreJob a.capabilities with

```

Comment:
Please normalize the indentation of this lambda.  Also please start each line of a pipeline with the pipeline operator at the same level of indentation as the head of the pipeline,
```fsharp
let providers =
    markerProviders
    |> List.choose (...)
```

Reply:
Also cleaner here to use Option.map rather than matching on the option as the None case passes through.